### PR TITLE
fix: remove fetchallpages saga

### LINKED ISSUE #16650: [Bug]: The Same events are displayed multiple times
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

The Same events are displayed multiple times beacuse same API is called 2 times in network
- [ ] when user edit the workspace name from general tab the 2 update events are added 
- [ ] when user change website name in general tab the unnecessary events are displayed 
- [ ] when user navigate to member tab from setting menu the 2 events are added 
- [ ] When user clones an app, and renames it, we get two events of the same description

### Steps To Reproduce

1. Login to application 
2. click on setting tab
3. now in general tab edit website placeholder
4. click on members&setting menu in home page
5. click on audit log
6. The same event is displayed 2 times

### Public Sample App

_No response_

### Version

http://ec2-65-2-149-212.ap-south-1.compute.amazonaws.com/applications

### PR DESCRIPTION
**Pull request by dipyamanbiswas07**
## Description

Currently the fetchallpages saga calls all the pages in the application in view mode along with a call to fetch the current page. This PR will remove the extra APIs being called.

#### PR fixes following issue(s)
Fixes #16650 

- Bug fix (non-breaking change which fixes an issue)
>
>
>
- [x] Manual
- [ ] JUnit
- [ ] Jest
- [ ] Cypress
>
>
#### Dev activity
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] PR is being merged under a feature flag

#### QA activity:
- [ ] [Speedbreak features](https://github.com/appsmithorg/TestSmith/wiki/Guidelines-for-test-plans#speedbreakers-) have been covered
- [ ] Test plan covers all impacted features and [areas of interest](https://github.com/appsmithorg/TestSmith/wiki/Guidelines-for-test-plans#areas-of-interest-)
- [ ] Test plan has been peer reviewed by project stakeholders and other QA members
- [ ] Manually tested functionality on DP
- [ ] We had an implementation alignment call with stakeholders post QA Round 2
- [ ] Cypress test cases have been added and approved by SDET/manual QA
- [ ] Added `Test Plan Approved` label after Cypress tests were reviewed
- [ ] Added `Test Plan Approved` label after JUnit tests were reviewed

### CODE REVIEW

**Review by ankitakinger — DISMISSED** [review]
LGTM

**Review by arunvjn — DISMISSED** [review]
Looks good

File: app/client/src/ee/sagas/PageSagas.tsx [hunk 1]
```diff
@@ -49,10 +48,6 @@ export default function* pageSagas() {
     takeLatest(ReduxActionTypes.DELETE_PAGE_INIT, deletePageSaga),
     debounce(500, ReduxActionTypes.SAVE_PAGE_INIT, savePageSaga),
     takeLatest(ReduxActionTypes.UPDATE_WIDGET_NAME_INIT, updateWidgetNameSaga),
-    takeLatest(
```
Comment (ankitakinger) [inline app/client/src/ee/sagas/PageSagas.tsx:52]: if the fetchAllPublishedPagesSaga is no longer used anywhere, we can also think of removing it in future.

Comment (dipyamanbiswas07) [reply app/client/src/ee/sagas/PageSagas.tsx:52]: Removed.

**Review by ankitakinger — DISMISSED** [review]

**Review by ankitakinger — APPROVED** [review]

**Pull request closed without merging**